packages/core/src/controllers/events.ts

```diff
@@ -39,6 +39,32 @@ export class EventClient extends IEventClient {
     );
   }
 
+  public init: IEventClient["init"] = async () => {
+    if (typeof process !== "undefined" && process.env.IS_VITEST === "true") return;
+    try {
+      const initEvent = {
+        eventId: uuidv4(),
+        bundleId: this.core.projectId || "",
+        timestamp: Date.now(),
+        props: {
+          event: "INIT",
+          type: "",
```

Comment:
cc @dnul if you're relying on the type here, what should the value be ?

Reply:
as per the specs https://www.notion.so/walletconnect/Walletkit-events-sdk-init-event-08b51439dd8b42c5bbe993c63d2ce54e?pvs=4#49045ac3ba8d4f36900a24047b9cc87b
```
val event: String = "INIT"
val type: String, // None
```